packages/block-library/src/home-link/block.json

```diff
@@ -0,0 +1,39 @@
+{
+	"apiVersion": 2,
+	"name": "core/home-link",
+	"category": "design",
+	"parent": [ "core/navigation" ],
+	"title": "Home Link",
+	"keywords": [ "home", "menu", "navigation" ],
+	"textdomain": "default",
+	"attributes": {
+		"label": {
+			"type": "string"
+		},
+		"opensInNewTab": {
+			"type": "boolean",
+			"default": false
+		},
+		"rel": {
```

Comment:
What would be some use cases for `rel` on a home link?

Reply:
🤔 browsing https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types it doesn't seem like there is anything relevant. I can remove this for v1, and we can see if folks ask to add it.

Reply:
So I looked a bit and site title has `rel="home"` but there isn't any meaning to this in HTML5 spec. Is there a WP specific case for this?
 https://github.com/WordPress/gutenberg/blob/0ea3bcc05e33cfdc5a5945a0e408464357b73d87/packages/block-library/src/site-title/index.php#L23